OpenEdXMobile/res/values/strings.xml

```diff
@@ -604,5 +606,9 @@
     <string name="share_accomplishment_button">Share your accomplishment</string>
     <!-- Text that pre-populates your post/email/SMS when sharing certificate -->
     <string name="share_accomplishment_message">I just earned a badge on {platform_name}! Check it out: {badge_url}</string>
-
+    <string name="rating_dialog_message">How would you rate the edX app?</string>
+    <string name="rate_app_dialog_title">Rate the app</string>
+    <string name="rate_app_dialog_message">Tell others what you think of the edx app by writing a quick review in the app store</string>
```

Comment:
> Also, please fix - in the "rate_app_dialog_message" string, "edx" needs an uppercase "X".

Thinking in terms of white-labelling terms I think instead of "edX", we should use whatever the platform_name is instead of "edX" here.